includes/admin/class-wcmypa-admin.php

```diff
@@ -917,6 +931,116 @@ private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     }
 
     /**
+     * Output the chosen delivery options or the chosen pickup options.
+     *
+     * @param DeliveryOptions $deliveryOptions
+     *
+     * @return array[]|null
+     * @throws \Exception
+     */
+    private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
+    {
+        $signatureTitle     = (new WCMP_Checkout)->getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_SIGNATURE_TITLE);
+        $onlyRecipientTitle = (new WCMP_Checkout)->getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_ONLY_RECIPIENT_TITLE);
+
+        if (! $deliveryOptions->getCarrier()) {
+            return null;
+        }
+
+        if ($deliveryOptions->getDeliveryType() === AbstractConsignment::DELIVERY_TYPE_PICKUP_NAME) {
```

Comment:
Yoda comparisons (dus constante eerst) 

Als je het hier niet mee eens bent hoor ik het graag maar het is een standaard die gebruikt wordt in de API :)